client/app.jsx

```diff
@@ -1,8 +1,40 @@
 import React from 'react';
+import { parseRoute } from '../../c0621-code-solutions/react-hash-routing/client/lib';
```

Comment:
Please add the file to your project repository instead of pulling it in from a different repository.